tests/test_volume.py

```diff
@@ -726,6 +734,77 @@ def test_symmetry_group_set_get(sym_group, sym_string):
         _ = Volume(data, symmetry_group=123, dtype=dtype)
 
 
+def test_symmetry_group_pass_through(symmetric_vols):
+    vol_c3, _ = symmetric_vols
+    sym_group = str(vol_c3.symmetry_group)
+    assert sym_group == "C3"
+
+    # Check symmetry_group pass-through for various transformations.
+    assert str(vol_c3.astype(np.float64).symmetry_group) == sym_group  # astype
+    assert str(vol_c3[0].symmetry_group) == sym_group  # getitem
+    assert (
+        str(vol_c3.stack_reshape((1, 1)).symmetry_group) == sym_group
+    )  # stack_reshape
+    assert (
+        str(vol_c3.downsample(vol_c3.resolution // 2).symmetry_group) == sym_group
+    )  # downsample
+
+
+def test_transformation_symmetry_warnings(symmetric_vols):
+    """
+    A warning should be emitted (once) for transpose, flip, rotate, add, sub, mult, div.
+    """
+    vol_c3, _ = symmetric_vols
+    sym_group = str(vol_c3.symmetry_group)
+    assert sym_group == "C3"
+
+    # Check we get warning on first transformation.
```

Comment:
Not sure I follow this comment.  Is the code below testing for 3 warnings?

Reply:
Changed this comment. Yes, we are expecting 3 warnings here.